osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/application/services/ConfigurationService.java

```diff
@@ -907,17 +889,37 @@ public void handleGetMbusEncryptionKeyStatusByChannelResponse(final DeviceMessag
         LOGGER.info("handleGetMbusEncryptionKeyStatusByChannelResponse for MessageType: {}",
                 deviceMessageMetadata.getMessageType());
 
-        final EncryptionKeyStatusType encryptionKeyStatusType = EncryptionKeyStatusType
-                .valueOf(getMbusEncryptionKeyStatusByChannelResponseDto.getEncryptionKeyStatus().name());
-
-        ResponseMessage responseMessage = ResponseMessage.newResponseMessageBuilder()
-                .withCorrelationUid(deviceMessageMetadata.getCorrelationUid())
-                .withOrganisationIdentification(deviceMessageMetadata.getOrganisationIdentification())
-                .withDeviceIdentification(deviceMessageMetadata.getDeviceIdentification())
-                .withResult(resultType).withOsgpException(exception).withDataObject(encryptionKeyStatusType)
-                .withMessagePriority(deviceMessageMetadata.getMessagePriority()).build();
-        this.webServiceResponseMessageSender.send(
-                responseMessage,
-                deviceMessageMetadata.getMessageType());
+        final EncryptionKeyStatusType encryptionKeyStatusType = EncryptionKeyStatusType.valueOf(
+                getMbusEncryptionKeyStatusByChannelResponseDto.getEncryptionKeyStatus().name());
+
+        ResponseMessage responseMessage = ResponseMessage.newResponseMessageBuilder().withCorrelationUid(
+                deviceMessageMetadata.getCorrelationUid()).withOrganisationIdentification(
+                deviceMessageMetadata.getOrganisationIdentification()).withDeviceIdentification(
+                deviceMessageMetadata.getDeviceIdentification()).withResult(resultType).withOsgpException(
+                exception).withDataObject(encryptionKeyStatusType).withMessagePriority(
+                deviceMessageMetadata.getMessagePriority()).build();
+        this.webServiceResponseMessageSender.send(responseMessage, deviceMessageMetadata.getMessageType());
+    }
+
+    public void setRandomizationSettings(final DeviceMessageMetadata deviceMessageMetadata,
+            final org.opensmartgridplatform.domain.core.valueobjects.smartmetering.SetRandomizationSettingsRequestData setRandomizationSettingsRequestData)
+            throws FunctionalException {
+
+        LOGGER.info("setRandomizationSettings for organisationIdentification: {} for deviceIdentification: {}",
+                deviceMessageMetadata.getOrganisationIdentification(), deviceMessageMetadata.getDeviceIdentification());
+
+        //        final SmartMeter smartMeteringDevice = this.domainHelperService
+        //                .findSmartMeter(deviceMessageMetadata.getDeviceIdentification());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)